public/500.svg

```diff
@@ -0,0 +1,553 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
We shouldn't put all the `error` images into the root of the public folder. To be more organized, they should be placed into a folder of their own. This will make it easier for others to find the `error` images in the future